### PR TITLE
Added pub-sub alias

### PR DESCRIPTION
**Pull request by rrelledge**
The Pub/Sub link on the [About](https://redis.io/docs/about/) page redirected to /docs/manual/pub-sub instead of /docs/manual/pubsub, which caused a 404. This new alias should fix the 404.

### CODE REVIEW

**Comment by zuiderkwast** [comment]
Good idea! We should have redirects for all the old URIs. There are many broken links now.

**Comment by zuiderkwast** [comment]
I'm not sure the new-structure branch is used anymore. Perhaps you know this better than I do?

**Comment by rrelledge** [comment]
> I'm not sure the new-structure branch is used anymore. Perhaps you know this better than I do?

@zuiderkwast No, I think you are right. Back when I opened this PR, I think master might have been slightly out-of-sync with the new-structure branch, so I wasn't sure. I can try to change the destination branch for this PR to master. But if that might cause issues, we can close the PR without merging and open a new one that points to correct branch.

**Comment by rrelledge** [comment]
@zuiderkwast Done. It looks like changing the destination worked okay to me.

**Review by zuiderkwast — APPROVED** [review]

**Comment by rrelledge** [comment]
@zuiderkwast By the way, I can't merge this PR myself since I don't have write access.

**Pull request merged**
